### DISCUSSION TITLE
2021-07-28 Weekly Meeting 14 Agenda

### DISCUSSION THREAD

**Discussion by humbletim** [omigroup/omigroup#66] [Weekly Meetings]
Weekly Meeting 14 is happening on Wednesday 2021-07-28 from 9:00 AM PDT - 10:00 AM PDT

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR Discord](https://discord.gg/NJtT9grz5E).

## Agenda
- OMI Weekly Meeting Discord Bot touch base #63
- Website effort touchbase #49 
   - :calendar: Shooting for Thursday 7/29 at 5pm PDT 
- OMI Space to Meet, work and play #44
- GLTF extensions working group touch base #64
   - Pull Request 1 [OMI_audio_emitter #1](https://github.com/omigroup/gltf-extensions/pull/1)
   - follow-up from 7/22 open discussion
   - :calendar: Next meeting scheduled for Thursday 7/29 at 2:30pm PDT 
     - Workshops: Who are we serving? [gltf extensions #3](https://github.com/omigroup/gltf-extensions/issues/3)
- Fiscal Development Group touch base #57

## Add Ons
*To add an agenda item please comment below*

**Comment by mrmetaverse** [comment]
The fiscal development working group met last week and the demo days came up, with a potential first date, and some early information which is summarized [here](https://github.com/omigroup/OMI/issues/59#issuecomment-885216344)

Reminder that we have Issue #59 : We are hoping to form an early group to plan what that first demo day might look like, how often we would meet, etc. Demo days are different than lightning talks, in that it is our chance to share our work with each other. As we continue to grow, it will become increasingly difficult to discuss all work being conducted in the allotted weekly sync timeframe. A system of regular demo days would help provide a sync for breakthroughs and challenges, and our weekly syncs can continue to be high level outcomes, and objectives. :dart: 

We have two working session scheduled for this Thursday already (I believe) so I'd like to suggest that we meet either this Friday or just after our weekly sync this Wednesday, to outline what we think a lightweight, and functional demo day process would look like. 

Other events, like hackathons, lightning talks, and events may require more unique planning, whereas this is our chance to make demo days slightly more systematic and drive both discovery and delivery backlogs from those sessions.

**Comment by mrmetaverse** [comment]
I propose that we discuss #68 : Defining our working agreement/playbook from our backlog around drafting a team working playbook (an agile working agreement). I understand that some may think that sounds like a legal document :sweat_smile: and I assure you, it's not. But they are important documents for teams working together. 

Working Playbooks/Agreements help us craft living documents that highlight where work happens, how we respect each other, how we prefer to be engaged, where we will escilate issues, etc. A really great format can be found here, and I am happy to faciltate any teams requiring a working agreement session. I recommend we create one for each working group / team, as preferences change based on team makeup.

**Comment by robertlong** [comment]
I've got a late addition for the gltf-extensions topic: https://github.com/omigroup/three-omi

I've created a reference implementation of the OMI_audio_emitter proposal and set up this repository so we can quickly develop reference implementations of additional extensions. We could do the same for Godot or any other engine, or we can talk about other homes for these sorts of projects.

**Reply by mrmetaverse** [reply]
That's awesome! We have a baseline for iteration 🙌

**Comment by mrmetaverse** [comment]
Got the notes started early. If you hear something of interest, add a bullet and your note! https://hackmd.io/@mrmetaverse/omi-072821/edit
